src/game/clients/CClientTarg.cpp

```diff
@@ -652,6 +652,17 @@ bool CClient::OnTarg_Tile( CObjBase * pObj, const CPointMap & pt )
 
 				if ( m_Targ_Text.IsEmpty())
 				{
+					if (pItem->IsTypeMulti())
+					{
+						CItemMulti* pMulti = static_cast<CItemMulti*>(pItem);
```

Comment:
This could be redundant if the check was moved into citem::delete

Reply:
yes, can be done at to delete(), I prefer it as well.